actix-rt/src/lib.rs

```diff
@@ -206,3 +206,20 @@ where
 {
     tokio::task::spawn_local(f)
 }
+
+/// Runs the provided future, blocking the current thread until the future completes.
+///
+/// This is a shortcut for creating a [`System`] and calling [`System::block_on`].
```

Comment:
I think it'll be helpful to add whether this method will panic or not if called within the scope of another `System`, similarly to [`tokio::Handle::block_on`](https://docs.rs/tokio/latest/tokio/runtime/struct.Handle.html#method.block_on).